apps/web/src/features/admin/users/usersQueries.ts

```diff
@@ -0,0 +1,48 @@
+import { keepPreviousData, useQuery } from '@tanstack/react-query';
+import { getUsers, getUserById } from '@/features/admin/users/usersFetchers';
+import { MRT_PaginationState, MRT_SortingState } from 'material-react-table';
+
+export const useGetUsers = (
+  globalFilter: string,
+  pagination: MRT_PaginationState,
+  sorting: MRT_SortingState,
+) => {
+  return useQuery({
+    queryKey: [
+      'users',
+      globalFilter,
+      pagination.pageIndex,
+      pagination.pageSize,
+      sorting,
+    ],
+    queryFn: async () => {
+      let sortBy = 'name';
```

Comment:
check this, on backend default sort is email but on frontend default sort is name